metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -24,6 +24,28 @@
     CorpUserSnapshotClass,
 )
 
+# default mapping for attrs
+# general attrs
+attrs_mapping: Dict[str, Any] = {}
+attrs_mapping["urn"] = "sAMAccountName"
```

Comment:
So this is the attribute used to construct the urn?

Reply:
yes, thanks for clarifying that in the docs!